api/python/tests/integration/test_packages.py

```diff
@@ -18,6 +20,44 @@ def mock_make_api_call(operation_name):
         return parsed_response
     raise NotImplementedError
 
+@patch('appdirs.user_data_dir', side_effect=(lambda x,y: os.path.join('test_appdir', x)))
```

Comment:
`side_effect=` doesn't seem right to me... Just this should work:
```
@patch('appdirs.user_data_dir', lambda x,y: os.path.join('test_appdir', x))
```
But doesn't really matter, I guess